src/tests/vtkh/CMakeLists.txt

```diff
@@ -47,13 +47,26 @@
 # Unit Tests
 ################################
 
+set(TESTS_SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR})
+configure_file(
+  "${CMAKE_CURRENT_SOURCE_DIR}/t_vtk-h_contour_tree.cpp.in"
+  "${CMAKE_CURRENT_BINARY_DIR}/t_vtk-h_contour_tree_par.cpp")
+configure_file(
+  "${CMAKE_CURRENT_SOURCE_DIR}/t_vtk-h_contour_tree.cpp.in"
+  "${CMAKE_CURRENT_BINARY_DIR}/t_vtk-h_contour_tree.cpp")
+
```

Comment:
avoid configuring the test source, instead we can put the data file in a known place